perllib/FixMyStreet/App/Form/Waste/AboutYou.pm

```diff
@@ -62,6 +62,14 @@ has_field email => (
     },
 );
 
+has_field extra_bulky_text_updates => (
+    type => 'Checkbox',
+    label => 'Bulky text updates',
```

Comment:
I still think the inclusion of 'updates' here is misleading as the message aren't really updates - they're reminders.